### DISCUSSION TITLE
how to use font awesome pro sharp with QIcon

### DISCUSSION THREAD

**Discussion by charles-lebrun** [quasarframework/quasar#15494] [General - Components / Directives / etc]
I'm trying to use new sharp font awesome pro,

but it's seems that the new patterns dont match regex

exemple in font awesome pro website

`
<font-awesome-icon icon="fa-sharp fa-solid fa-box-archive" />
`

what I have test

```
<q-icon name="fa-sharp fa-solid fa-box-archive" size="lg"></q-icon>
<q-icon name="fass fa-box-archive" size="lg"></q-icon>
```

Could someone help me ?

**Comment by MilosPaunovic** [comment]
Have you followed the guide below?
https://quasar.dev/options/installing-icon-libraries#using-fontawesome-pro

**Reply by charles-lebrun** [reply]
Hello,

yes I follow this guide and work well for `classic` font awesome pro (solid, regular, duotone, light, thin).
since january Font awesome add new style `sharp` with sharp-regular and sharp-solid option

To use `sharp` we have to add a third class in the icon name, i think the component dont manage it

**Comment by charles-lebrun** [comment]
Does someone have any idea or can confirm that is an issue ?
